android/src/com/morpheme/palmpiano/util/Constants.java

```diff
@@ -28,4 +26,8 @@
         MODE_PLAYBACK,
         MODE_GAME
     }
+
+    // Shift the camera X position on initialize so it is around C3/ C4 on the keyboard
+    public static final int CAMERA_VIEWPORT_X_OFFSET = 2000;
+    public static final String localPath = "/data/user/0/com.morpheme.palmpiano/files/midi/";
```

Comment:
If we're going to have localPath here (which I think is a good idea), should we also add the internal/assets path here as well?